src/XMakeBuildEngine/Construction/Solution/ProjectInSolution.cs

```diff
@@ -22,12 +22,30 @@ namespace Microsoft.Build.Construction
     /// </remarks>
     public enum SolutionProjectType
     {
-        Unknown,            // Everything else besides the below well-known project types.
-        KnownToBeMSBuildFormat,     // C#, VB, and VJ# projects
-        SolutionFolder,     // Not really a project, but persisted as such in the .SLN file.
-        WebProject,         // Venus projects
-        WebDeploymentProject, // Web Deployment (.wdproj) projects -- MSBuildFormat, but Whidbey-era ones specify ProjectReferences differently
-        EtpSubProject       // Project inside an Enterprise Template project
+        /// <summary>
+        /// Everything else besides the below well-known project types.
+        /// </summary>
+        Unknown,
+        /// <summary>
+        /// C#, VB, and VJ# projects
+        /// </summary>
+        KnownToBeMSBuildFormat,
+        /// <summary>
+        /// 
+        /// </summary>
+        SolutionFolder,       // Not really a project, but persisted as such in the .SLN file.
```

Comment:
The comment seemed really "internal" and not something we'd want showing up in intellisense.  Do you want to reword it to sound more customer-facing?